scripts/reconcile-state.sh

```
@@ -0,0 +1,344 @@
+check_worktrees_exist() {
+    local worktree_count
+    worktree_count="$(jq '.worktrees | length' "$STATE_FILE")"
+
+    if [[ "$worktree_count" -eq 0 ]]; then
+        check_pass "Worktrees exist (no worktrees to check)"
+        return 0
+    fi
+
+    # Get actual git worktrees
+    local git_worktrees
+    git_worktrees="$(git -C "$REPO_ROOT" worktree list --porcelain 2>/dev/null | grep '^worktree ' | sed 's/^worktree //' || true)"
```

Shellcheck (0.11.0)

[warning] git_worktrees appears unused. Verify use (or export if used externally).

(SC2034)